README.md

```diff
@@ -101,6 +101,10 @@ If you want to customize the application environment, reference this table:
 | REDIS_SSL_ENABLED                                        | Whether SSL is enabled while connecting to  Redis                  | string                   |         |
 | WALLET_SESSION_TTL                                       | Wallet session TTL in minutes                                      | int                      |         |
 | WALLET_LOGO_MAPPING                                      | Wallet logo map parameter that handles wallet type to logo mapping | string (Map<string,URI>) |         |
+| EXPIRATION_QUEUE_NAME                                    | Name of expiration queue                                           | string                   |         |
+| EXPIRATION_QUEUE_TTL_SECONDS                             | TTL in seconds for published message                               | string                   |         |
+| EXPIRATION_QUEUE_CONNECTION_STRING                       | Connection string to storage queue                                 | string                   |         |
+| EXPIRATION_QUEUE_TIMEOUT_SECONDS_WALLET_EXPIRED          | Visibility timeout in seconds for expired event                    |                          |         |
```

Comment:
what about rename this to `EXPIRATION_QUEUE_VISIBILITY_TIMEOUT_SEC`?

Reply:
Solved by https://github.com/pagopa/pagopa-wallet-service/pull/148/commits/dcd30dd8720e5e8f06681487beec27ecc1bf1b43